code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -73,7 +100,9 @@
 	ADD_TRAIT(owner, TRAIT_TURRET_HIDDEN, STEALTH_TRAIT)
 
 	handle_stealth()
-	addtimer(CALLBACK(src, PROC_REF(sneak_attack_cooldown)), HUNTER_POUNCE_SNEAKATTACK_DELAY) //Short delay before we can sneak attack.
+	addtimer(CALLBACK(src, PROC_REF(sneak_attack_cooldown)), HUNTER_POUNCE_SNEAKATTACK_DELAY)
+	if(stealth_duration != -1)
```

Comment:
Does make a practical difference, but better to just have stealth duration be null, and have this if statement just check for the var in general.

Reply:
fair i thought having it a number would prevent issues but it works fine with null too looks like